cloudevents/extensions/jwt.md

```diff
@@ -0,0 +1,147 @@
+# JWT Extension
+
+This extension provides a way to map JWT (JSON Web Token) values in the formats of JWE
+ (JSON Web Encryption) or JWS (JSON Web Signature).
```

Comment:
might want to add " to CloudEvents context attributes" to the end of the sentence. Otherwise we're missing what we're mapping the JWT values into.